### PR TITLE
[vcpkg docs] Add the procedure for updating vcpkg to faq

### LINKED ISSUE #15417: How do I update vcpkg itself?
I looked at the FAQ of "readthedocs" and I can't find a clear way of updating vcpkg itself. Can someone tell me the procedure?

### PR DESCRIPTION
**Pull request by PhoebeHui**
Fixes https://github.com/microsoft/vcpkg/issues/15417

I thinks it's fine to add the information to faq.

### CODE REVIEW

File: docs/about/faq.md [hunk 1]
```diff
@@ -30,6 +30,9 @@ We currently target Windows Desktop (x86 and x64) as well as the Universal Windo
 ## Does Vcpkg run on Linux/OSX?
 Yes! We continuously test on OSX and Ubuntu 16.04, however we know users have been successful with Arch, Fedora, and FreeBSD. If you have trouble with your favorite Linux distribution, let us know in an issue and we'd be happy to help!
 
+## How do I update vcpkg itself?
+You can execute `git pull` to get latest source, then runing bootstrap-vcpkg.bat(on Windows) or bootstrap-vcpkg.sh(on Unix) to update vcpkg.
```
Comment (autoantwort) [inline docs/about/faq.md:34]:
```suggestion
You can execute `git pull` to get latest source, then runing `bootstrap-vcpkg.bat` (Windows) or `./bootstrap-vcpkg.sh` (Unix) to update vcpkg.
```

Comment (dg0yt) [inline docs/about/faq.md:34]: "... then *run* ..."

**Comment by PhoebeHui** [comment]
@autoantwort @dg0yt, Very appreciate for your review!

**Comment by JackBoosY** [comment]
depends on #17277

File: docs/about/faq.md [hunk 2]
```diff
@@ -30,6 +30,9 @@ We currently target Windows Desktop (x86 and x64) as well as the Universal Windo
 ## Does Vcpkg run on Linux/OSX?
 Yes! We continuously test on OSX and Ubuntu 16.04, however we know users have been successful with Arch, Fedora, and FreeBSD. If you have trouble with your favorite Linux distribution, let us know in an issue and we'd be happy to help!
 
+## How do I update vcpkg itself?
```
Comment (Hoikas) [inline docs/about/faq.md:33]:
```suggestion
## How do I update vcpkg?
```

The word "itself" is redundant, IMO.

File: docs/about/faq.md [hunk 3]
```diff
@@ -30,6 +30,9 @@ We currently target Windows Desktop (x86 and x64) as well as the Universal Windo
...
+You can execute `git pull` to get latest source, then run `bootstrap-vcpkg.bat` (Windows) or `./bootstrap-vcpkg.sh` (Unix) to update vcpkg.
```
Comment (Hoikas) [inline docs/about/faq.md:34]:
```suggestion
Execute `git pull` to get the latest sources, then run `bootstrap-vcpkg.bat` (Windows) or `./bootstrap-vcpkg.sh` (Unix) to update vcpkg.
```

More active wording

File: docs/about/faq.md (hunk 2 above)
Comment (PhoebeHui) [reply docs/about/faq.md]: Updated, thanks for your review!

File: docs/about/faq.md (hunk 3 above)
Comment (PhoebeHui) [reply docs/about/faq.md]: Updated, thanks!

**Review by JackBoosY — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for your contribution!

**Pull request merged**